packages/xo-web/src/common/intl/messages.js

```diff
@@ -498,6 +498,8 @@ const messages = {
   timeout: 'Timeout',
   timeoutInfo: 'Number of hours after which a job is considered failed',
   fullBackupInterval: 'Full backup interval',
+  fullBackupIntervalInfo:
+    'After which interval a full backup will be triggered',
```

Comment:
What is an *interval*?
When is after an *interval*?

Reply:
> What is an interval?
It's the full backup interval var set by the user
> When is after an interval?
After which execution interval a full backup will be triggered?

Reply:
What I meant is that your explanation is unclear, is the interval a duration?
Is it defined in hours or something else?

Don't explain it to me, explain it to the users.